mino/minows/address_test.go

```diff
@@ -9,65 +9,78 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-const (
-	AddrAllInterface = "/ip4/0.0.0.0/tcp/80"
-	AddrLocalhost    = "/ip4/127.0.0.1/tcp/80"
-	AddrHostname     = "/dns4/example.com/tcp/80"
-)
-const (
-	PID1 = "QmaD31nEzFGwD8dK96UFWHtTYTqYJgHLMYSFz4W4Hm2WCU"
-	PID2 = "QmVt9t5Tk2uEoA4CDbKNCVxqrut8UXmWHXvFZ8wFZ3ghhv"
-)
-
-func TestNewAddress(t *testing.T) {
+func Test_newAddress(t *testing.T) {
+	const pid1 = "QmaD31nEzFGwD8dK96UFWHtTYTqYJgHLMYSFz4W4Hm2WCU"
+	const pid2 = "QmVt9t5Tk2uEoA4CDbKNCVxqrut8UXmWHXvFZ8wFZ3ghhv"
+	const addrAllInterface = "/ip4/0.0.0.0/tcp/80"
+	const addrLocalhost = "/ip4/127.0.0.1/tcp/80"
+	const addrHostname = "/dns4/example.com/tcp/80"
+	type args struct {
+		location string
+		identity string
+	}
 	tests := map[string]struct {
-		location ma.Multiaddr
-		identity peer.ID
-		err      bool
+		args args
 	}{
-		"missing location": {
-			location: nil,
-			identity: mustCreatePeerID(PID1),
-			err:      true,
-		},
-		"missing identity": {
-			location: mustCreateMultiaddress(AddrAllInterface),
-			identity: "",
-			err:      true,
-		},
 		"all interface": {
-			location: mustCreateMultiaddress(AddrAllInterface),
-			identity: mustCreatePeerID(PID1),
+			args: args{addrAllInterface, pid1},
 		},
 		"localhost": {
-			location: mustCreateMultiaddress(AddrLocalhost),
-			identity: mustCreatePeerID(
-				PID2),
+			args: args{addrLocalhost, pid2},
 		},
 		"hostname": {
-			location: mustCreateMultiaddress(AddrHostname),
-			identity: mustCreatePeerID(
-				PID1),
+			args: args{addrHostname, pid1},
 		},
 	}
+	t.Parallel() // run this test function args parallel to other test functions
```

Comment:
Here and elsewhere: the dela guid asks comments to be on their own line. So this should be:
```suggestion
    // run this test function args parallel to other test functions
	t.Parallel() 
```

And commenting obvious things should be avoided, like this line.

Reply:
moved to own lines. maybe it's not obvious at first glance, since the two parallel calls are parallelizing diff things?

Reply:
Yes, true. I was confused by the fact that tests from multiple packages are run in parallel, but the tests within one package are run sequentially:

https://bryce.is/writing/code/go-test-and-parallelism

(Which also explains the PAUSE we saw in one of the test runs)

Again, for the unity of the code, I would not run the tests in parallel here. Or only if you have a really good reason, like running the tests 10'000 times or so.

I try to avoid surprises like these, for debugging's sake. As all the other tests in `dela` are sequential, please make these tests also sequential.